### PR TITLE
Run CI on desktop tests again

### PR DESCRIPTION
**Pull request by jdlrobson**
I temporarily switched Pixel over to the mobile tests to unblock CI.
I am not sure what's wrong with CI in desktop but am hoping  we'll be able to replicate this on https://pixel.wmcloud.org/ at least!

### CODE REVIEW

**Comment by jdlrobson** [comment]
@moabualruz  Failures likely have something to do with:
```
ProtocolError: Protocol error (Runtime.callFunctionOn): Target closed.
    at /usr/local/lib/node_modules/backstopjs/node_modules/puppeteer/lib/cjs/puppeteer/common/Connection.js:278:24
    at new Promise (<anonymous>)
    at CDPSession.send (/usr/local/lib/node_modules/backstopjs/node_modules/puppeteer/lib/cjs/puppeteer/common/Connection.js:274:16)
    at ExecutionContext._ExecutionContext_evaluate (/usr/local/lib/node_modules/backstopjs/node_modules/puppeteer/lib/cjs/puppeteer/common/ExecutionContext.js:268:46)
    at ExecutionContext.evaluateHandle (/usr/local/lib/node_modules/backstopjs/node_modules/puppeteer/lib/cjs/puppeteer/common/ExecutionContext.js:162:107)
    at WaitTask.rerun (/usr/local/lib/node_modules/backstopjs/node_modules/puppeteer/lib/cjs/puppeteer/common/DOMWorld.js:664:37)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
```

File: src/engine-scripts/puppet/scroll.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 module.exports = async ( page ) => {
 	await page.evaluate( () => {
-		window.scroll( 0, window.innerHeight * 2 );
+		window.scroll( 0, 1500 );
```
Comment (moabualruz) [inline src/engine-scripts/puppet/scroll.js:3]: the old value was

```suggestion
		window.scroll( 0, window.innerHeight );
```
do we want to revert it to the old value or 1500 is fine

**Review by moabualruz — CHANGES_REQUESTED** [review]

**Comment by moabualruz** [comment]
With client preferences the toggle process is different this needs updating, this is the unfound target

File: LocalSettings.php [hunk 2]
```diff
@@ -385,3 +385,8 @@
 
 $wgMFAmcOutreachMinEditCount = 0;
 $wgMFAmcOutreach = true;
+$wgVectorZebraDesign = [
+	"logged_in" => true,
+	"logged_out" => true,
+	"beta" => true,
+];
```
Comment (moabualruz) [inline LocalSettings.php:392]:
```suggestion
```

**Review by moabualruz — APPROVED** [review]

**Pull request closed without merging**